composer.json

```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": "^5.6 || ^7.0",
         "container-interop/container-interop": "^1.0",
-        "pauci/cqrs": "^0.1"
+        "pauci/cqrs": "^0.2 || dev-master"
```

Comment:
Is that dev-master necessary? I would rather rely only on marked "stable" versions.